src/resources/Catalogs/Catalog.ts

```diff
@@ -50,4 +52,10 @@ export default class Catalog extends Resource {
     getFieldsSuggestions(query: FieldsSuggestionsQueryModel) {
         return this.api.post<FieldsSuggestionsModel>(`${Catalog.baseUrl}/fieldsSuggestions`, query);
     }
+
+    getFieldStats(catalogId: string, options?: CatalogFieldStatsOptions) {
+        return this.api.get<CachedCatalogFieldStatsModel>(
```

Comment:
Are you sure that's the right type, 
In [swagger](https://platformdev.cloud.coveo.com/docs?urls.primaryName=Catalog#/Catalogs/rest_organizations_paramId_catalogs_paramId_fieldStats_get) I see a different return type
```
[
  {
    "availability": {
      "objectsWithField": 0,
      "objectsWithType": 0
    },
    "fieldName": "string",
    "product": {
      "objectsWithField": 0,
      "objectsWithType": 0
    },
    "variant": {
      "objectsWithField": 0,
      "objectsWithType": 0
    }
  }
]
```

Reply:
Yes. Wrong version. I've updated the return type.